mod-audit-server/src/main/java/org/folio/dao/acquisition/impl/InvoiceLineEventsDaoImpl.java

```diff
@@ -0,0 +1,115 @@
+package org.folio.dao.acquisition.impl;
+
+import static java.lang.String.format;
+import static org.folio.util.AuditEventDBConstants.ACTION_DATE_FIELD;
+import static org.folio.util.AuditEventDBConstants.ACTION_FIELD;
+import static org.folio.util.AuditEventDBConstants.EVENT_DATE_FIELD;
+import static org.folio.util.AuditEventDBConstants.ID_FIELD;
+import static org.folio.util.AuditEventDBConstants.INVOICE_ID_FIELD;
+import static org.folio.util.AuditEventDBConstants.INVOICE_LINE_ID_FIELD;
+import static org.folio.util.AuditEventDBConstants.MODIFIED_CONTENT_FIELD;
+import static org.folio.util.AuditEventDBConstants.ORDER_BY_PATTERN;
+import static org.folio.util.AuditEventDBConstants.TOTAL_RECORDS_FIELD;
+import static org.folio.util.AuditEventDBConstants.USER_ID_FIELD;
+import static org.folio.util.DbUtils.formatDBTableName;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.util.Date;
+import java.util.UUID;
+
+import io.vertx.core.Future;
+import io.vertx.core.json.JsonObject;
+import io.vertx.sqlclient.Row;
+import io.vertx.sqlclient.RowSet;
+import io.vertx.sqlclient.Tuple;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.dao.acquisition.InvoiceLineEventsDao;
+import org.folio.rest.jaxrs.model.InvoiceLineAuditEvent;
+import org.folio.rest.jaxrs.model.InvoiceLineAuditEventCollection;
+import org.folio.util.PostgresClientFactory;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class InvoiceLineEventsDaoImpl implements InvoiceLineEventsDao {
+
+  private static final Logger LOGGER = LogManager.getLogger();
```

Comment:
You can use lombok to generate loggers in here and other classes too @Log4j2

Reply:
> You can use lombok to generate loggers in here and other classes too `@Log4j2`

This module like mod-orders may fail with lombok annotations on CI build without some additional pom.xml build settings